### PR TITLE
Mining economy changes

### PR DESCRIPTION
**Pull request by ghost**
- INCREASED RATES FOR ALL ORES IN ASTEROID ROCK
- LOWERED PRICES FOR MANY BASIC TOOLS/ITEMS/BULLETS/CIRCUIT BOARDS
- INCREASED RARE ORE PRICES FOR SOME THINGS IN THE PROTOLATHE (WEAPONS MOSTLY)
- PLASMA PLAYS A LITTLE BIT OF A LARGER ROLE IN SOME OF THE PROTOLATHE THINGS

THIS IS BASED ON ERGO'S PR TO /TG/

ALL CAPS

### CODE REVIEW

**Comment by ghost** [comment]
THIRD COMMIT ALSO KNOWN AS PAPRIKA IS BAD AT CODING AND BYOND LIES

File: code/modules/mining/mine_turfs.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	blocks_air = 1
 	temperature = TCMB
 	var/mineralName = ""
-	var/mineralAmt = 0
+	var/mineralAmt = 3
```
Comment (ChangelingRain) [inline code/modules/mining/mine_turfs.dm:14]: they all manually set the mineral amount to 5, so you have to remove those or miners will be getting an extra two sheets per deposit

File: code/modules/research/designs.dm [hunk 2]
```diff
@@ -1633,7 +1631,7 @@ datum/design/temp_gun
 	id = "temp_gun"
 	req_tech = list("combat" = 3, "materials" = 4, "powerstorage" = 3, "magnets" = 2)
 	build_type = PROTOLATHE
-	materials = list("$metal" = 5000, "$glass" = 500, "$silver" = 3000)
+	materials = list("$metal" = 10000, "$glass" = 1000, "$silver" = 5000)
```
Comment (ChangelingRain) [inline code/modules/research/designs.dm]: temp gun a shit, don't do this

File: code/modules/mining/mine_turfs.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mining/mine_turfs.dm:14]: I think that's intentional.

**Comment by GunHog** [comment]
Acceptable. :+1:

**Pull request merged**